### DISCUSSION TITLE
Ask for a windows start guide ToT

### DISCUSSION THREAD

**Discussion by Mming11** [praveen-palanisamy/macad-gym#78] [Q&A]
When i start on windows 10, there are some questions. So i hope to have a guide on windows.

**Comment by praveen-palanisamy** [comment]
Hi @Mming11 , The [Getting Started section in the README](https://github.com/praveen-palanisamy/macad-gym#getting-started) goes over the changes in the step needed to set it up on Windows. Except those minor changes, rest of the instructions are common between Linux and Windows because of the Python environment setup. 
Please try out and update your progress or ask again if you have any specific questions or step you need help with.

**Reply by Mming11** [reply]
Ok! Thank you. I have achieved it on Linux.
